pom.xml

```diff
@@ -64,11 +64,11 @@
   </scm>
 
   <properties>
-    <revision>2.12.4-2</revision>
+    <revision>2.12.4</revision>
```

Comment:
the second example on the page is the recommended format for library plugins:
https://www.jenkins.io/doc/developer/publishing/releasing-cd/#pom-file-modifications

Also changelist.format change seems to be missing here?

Reply:
should be fixed in 4af2a67a5327560e768133b9886b2de1ceb97c3b